tests/tensorflow/save_tf_estimator_model.py

```diff
@@ -0,0 +1,249 @@
+import sys
+
+if '' in sys.path:
+    # prevent importing mlflow in cwd.
+    sys.path.remove('')
```

Comment:
```suggestion
```

I think we can run this script in an empty temporary directory.

Reply:
> I think we can run this script in an empty temporary directory.

But it raises error (cannot find mlruns directory).
Because the subprocess reuse the Tracking URI of the parent testing process, but issue happen if the subprocess working directory different from parent process, seemingly relating to mlflow internally uses relative file path of `mlruns` directory

Reply:
I see, can we call `mlflow.set_tracking_uri` in the script?

Reply:
@harupy 

The tracking URI in mlflow test is like
```
sqlite:////private/var/folders/z7/47xgtbq53kg8hs1zn7qk8kw40000gp/T/pytest-of-weichen.xu/pytest-141/test_xxx0/mlruns.sqlite
```
But, for artifacts, it still uses local file path (relative path `./mlruns`). How to change it to be absolute path ?

Reply:
I think you can create a new experiment with an absolute artifact URI.